app/main/views/services.py

```diff
@@ -194,11 +208,17 @@ def import_service(service_id):
     service.updated_at = now
     service.created_at = now
     service.status = service_data.pop('status', 'published')
-    service.updated_by = updater_json['updated_by']
-    service.updated_reason = updater_json['update_reason']
     service.data = service_data
 
+    audit = AuditEvent(
+        audit_type=AuditTypes.import_service.value,
+        user=updater_json['updated_by'],
+        data=service_json,
+        db_object=None
```

Comment:
Done this, note that 
1) Audit events have to be made after flush

2) flush will throw the integrity check exceptions that normally happen on commit, so the whole session.add, session.flush, make audit event, session.add(audit) are now in the same try/catch as commit.